src/sqlparser.rs

```diff
@@ -1526,14 +1528,19 @@ impl Parser {
         Ok(SQLSelect {
             distinct,
             projection,
+            from,
             selection,
-            relation,
-            joins,
             group_by,
             having,
         })
     }
 
+    pub fn parse_table_reference(&mut self) -> Result<TableReference, ParserError> {
+        let relation = self.parse_table_factor()?;
+        let joins = self.parse_joins()?;
```

Comment:
Would `parse_joins` be useful elsewhere? Perhaps inline it here?

Reply:
Nope, not that I can see. Inlined!